src/lib/abacus/logger.ts

```diff
@@ -0,0 +1,12 @@
+import type { AbacusLoggingProtocol } from '@/constants/abacus';
+
+class AbacusLogger implements Omit<AbacusLoggingProtocol, '__doNotUseOrImplementIt'> {
+  d(tag: string, message: string) {
+    console.log(`${tag}: ${message}`);
+  }
+  e(tag: string, message: string) {
+    console.error(`${tag}: ${message}`);
+  }
```

Comment:
Is `Abacus`' `logger.d` aware of environment? We should skip depending on `dev/staging` vs `production`

Reply:
oh good point, I can reuse `VITE_ENABLE_ABACUS_LOGGING` to enable or disable it